lib/rubocop_director/commands/generate_config.rb

```diff
@@ -28,6 +28,8 @@ def run
       private
 
       def load_config
+        return Failure("#{RUBOCOP_DIRECTOR_CONFIG} already exists") if File.file?(RUBOCOP_DIRECTOR_CONFIG)
```

Comment:
I thought we will _ask_ user if he wants to override file, i.e.

```ruby
puts "#{RUBOCOP_DIRECTOR_CONFIG} already exists, do you want to override it? (y, n)"
if gets == 'y' 
```

Reply:
Okay, that makes sense.